x/feemarket/keeper/integration_test.go

```diff
@@ -234,9 +234,6 @@ var _ = Describe("Feemarket", func() {
 						// Note that max priority fee per gas can't be higher than the max fee per gas (gasFeeCap), i.e. 30_000_000_000)
 						return txParams{nil, big.NewInt(minGasPrices - 10_000_000_000), big.NewInt(30_000_000_000), &ethtypes.AccessList{}}
 					}),
-					Entry("dynamic tx with GasFeeCap > MinGasPrices, EffectivePrice < MinGasPrices", func() txParams {
```

Comment:
Is there any reason to delete?

Reply:
https://github.com/xpladev/ethermint/pull/14/commits/9a354f1e613aec65403d70ff899b758080d9060d